### DISCUSSION TITLE
Dev gone?

### DISCUSSION THREAD

**Discussion by DeadSecTr** [sheikhhaziq/gyawun_music#97] [General]
No updates anymore? Is this app dead?

**Comment by sheikhhaziq** [comment]
No, the app is not dead. Beta versions are available only on telegram and only stable versions will be uploaded on github.

**Comment by sheikhhaziq** [comment]
If you wanna checkout the latest version, join our telegram channel.

**Reply by DeadSecTr** [reply]
Ok thx
